### ISSUE TITLE
How do I add additional buttons on the message ?

### ISSUE THREAD

**Issue by Thilaknath** [jenkinsci/office-365-connector-plugin#258]
### Feature Request

Hello Team,  
I would like to learn, How I can control the templating of the message that is being notification that is sent out. By default the office365ConnectorSend step adds the "View Build" button along with links to the build.

![Screen Shot 2021-09-01 at 9 34 58 AM](https://user-images.githubusercontent.com/3452073/131680884-50c30510-f933-45b6-bcbf-49e63324892b.png)

I would like to further customize the above, by adding links to build artifacts and few other more resources to make the notification more meaningfull. 

I prefer adding them in a similar format (Button with Link) -> How can I achieve this?

**Comment by damianszczepanik** [comment]
There is no such option

**Comment by mlasevich** [comment]
Would also like to see an option to hide the "View Build" button - in some cases we have notifications for jobs people have no access to, but they still click on the button - because people :-D.  Ideally if we can expose the potential actions to be set, that would make things much simpler

**Comment by damianszczepanik** [comment]
> Hello and thank you for this connector slightly_smiling_face
> 
> And could it be possible to not display the button "View build"? It takes up a lot of space and might not be necessary, especially when the build is successful.

Sounds good. Here you are #272

**Comment by mlegait** [comment]
Thank you so much @damianszczepanik 🙏 

Is it in the last release 4.17.0?

**Comment by marcjay** [comment]
> Thank you so much @damianszczepanik 🙏
> 
> Is it in the last release 4.17.0?

@mlegait I just tested it, and yes it is. Also can see it in the diff: https://github.com/jenkinsci/office-365-connector-plugin/compare/Office-365-Connector-4.16.1...Office-365-Connector-4.17.0

**Comment by DarkGraySun** [comment]
This feature is bad. For me it is very important that unsuccessful builds are also displayed with the "View Build" button. I had to roll back to version 4.15. Please make sure that the "View Build" button is configurable.

**Comment by marcjay** [comment]
> For me it is very important that unsuccessful builds are also displayed with the "View Build" button
@DarkGraySun That is the case with the latest version, is it not? The button appears if the build is unsuccessful

**Comment by nsaadi** [comment]
For me it could be better if the "View Build" button is configurable, we want to use it if build successfull.

**Comment by martin-kirilov** [comment]
I agree this should be configurable, I'm also interested in having the "View Build" option on successful builds.
